### PR TITLE
httpd: formula tweaks.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- replace various `prefix` references with `opt_prefix` references
- put `htdocs` and `cgi-bin` back in `/var/www`.

CC @ilovezfs and @kabel for thoughts.

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
> put htdocs and cgi-bin back in /var/www.

Sticking with `var/www` to be consistent with `nginx` and `lighttpd` formulae.

File: Formula/httpd.rb [hunk 1]
```diff
@@ -52,10 +59,19 @@ def install
 
     # avoid using Cellar paths
     inreplace %W[
-      #{lib}/httpd/build/config_vars.mk
       #{include}/httpd/ap_config_layout.h
+      #{lib}/httpd/build/config_vars.mk
     ] do |s|
       s.gsub! "#{lib}/httpd/modules", "#{HOMEBREW_PREFIX}/lib/httpd/modules"
+    end
+    inreplace %W[
```
Comment (ilovezfs) [inline Formula/httpd.rb]: A blank line should be above this.

**Comment by kabel** [comment]
LGTM

**Comment by ilovezfs** [comment]
@kabel are /icons and /errors paths that can contain user data?

**Comment by kabel** [comment]
While technically they _could_, generally they do not. It all depends on what people decide to do with their config.

**Comment by kabel** [comment]
Now that I think about it more, we should probably move the `manualdir` to `pkgshare` too, as it definitely shouldn't contain user data.

**Comment by MikeMcQuaid** [comment]
> Now that I think about it more, we should probably move the manualdir to pkgshare too, as it definitely shouldn't contain user data.

@kabel Done, good idea.

File: Formula/httpd.rb (hunk 1 above)
Comment (MikeMcQuaid) [reply Formula/httpd.rb]: @ilovezfs Done (and for the one below)

**Review by ilovezfs — APPROVED** [review]

**Comment by edrozenberg** [comment]
Thanks for all these fixes.

I noticed ```DocumentRoot``` is now set to ```#{var}/www```, not the more "standard" ```#{var}/www/htdocs``` So after httpd install I see ```/usr/local/var/www/index.html``` rather than ```/usr/local/var/www/htdocs/index.html```

**Comment by ilovezfs** [comment]
@edrozenberg as noted in the comment above (https://github.com/Homebrew/homebrew-core/pull/18855#issuecomment-333623079), that's to be consistent with the nginx and lighttpd formulae. So we'd need to change those too in order to switch to the standard location.

**Comment by edrozenberg** [comment]
@ilovezfs thanks, wasn't sure the consistency comment was referring to this. Makes sense, easy change for me in httpd.conf anyway if I want it.

**Comment by ilovezfs** [comment]
@edrozenberg you're welcome!

**Pull request closed without merging**